src/library/simplified/extension/trip-type.js

```diff
@@ -55,7 +53,9 @@
     {
         b['tripType'] = ROUND_TRIP;
     }
-    else if ((utag.isFRateDetails() || utag.isPRateDetails()) && b['entity.tripDetails.flightOffer.tripType'] != undefined){
+    else if (utag.isPRateDetails() && b['entity.tripDetails.flightOffer.tripType'] != undefined){
         b['tripType'] = getTripType(b['entity.tripDetails.flightOffer.tripType']);
     }
-})();
\ No newline at end of file
+    else if (utag.isFRateDetails() && b['entity.tripDetails.flightOffer.tripType'] != undefined){
+        b['tripType'] = b['entity.tripDetails.flightOffer.tripType']
```

Comment:
Hi @trungnguyenexp ,
According to the doubleclick fire , the triptype (u20) looks like OW RT etc..
So I removed the getTripType()  call.
Sample fire:
doub
https://fls.doubleclick.net/activityi;src=2588797;type=nausc826;cat=nausr653;u1=Flight;u2=DEL;u3=SFO;u4=20170401|20170402;u7=3|0;u10=SQ|SQ;u11=Economy/Coach;u14=178.493|0.00;u15=1,606.44;u16=USD;u18=SQ31|SQ402;**u20=OW**;u=3d3f9656-9ea2-4f18-ae45-9f1597c80baf;ord=1;num=6312990565403.344?